DotNetRu.Clients.Portable/TweetHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LinqToTwitter;
+
+namespace XamarinEvolve.Clients.Portable
+{
+	public class TweetHelper
+	{
+		public static async Task<List<Tweet>> Get()
+		{
+			try
+			{
+			    var auth = new ApplicationOnlyAuthorizer
+			    {
+			        CredentialStore = new InMemoryCredentialStore
+			        {
+			            ConsumerKey = "t6tGj3XmaOzDQUnCIgih8gF4F",
```

Comment:
Is it OK to save API Key in public place? [Good article](http://kerry.lothrop.de/wedskaas1/)

Reply:
It's not OK, but let's keep it as it is for now.
This key provides read-only access, so it's not too dangerous. 
In future, let's move keys to our own backend,
BTW, thanks for the article!

Reply:
If this key provide read-only access, I don't have any more comments. I think #14 should be merged via squash